support-workers/src/test/scala/com/gu/zuora/ProductSubscriptionBuildersSpec.scala

```diff
@@ -94,10 +94,4 @@ class ProductSubscriptionBuildersSpec extends AnyFlatSpec with Matchers with Pro
     isTestUser = false,
     contractEffectiveDate = saleDate)
 
-  def checkAllowed(assertion: => Assertion) = {
```

Comment:
I was confused when I saw it above - it's gone now but probably should have called this `onlyIfFixedTermAllowed`